### PR TITLE
Feat ts dynamic import

### PR DESCRIPTION
**Pull request by akartis-dev**
Configure to change import to alias, and fix vitest.config.ts to avoid "ReferenceError: React is not defined" when executing test

### CODE REVIEW

**Comment by akartis-dev** [comment]
Relative to #5

File: vitest.config.ts [hunk 1]
```diff
@@ -7,4 +8,10 @@ export default defineConfig({
     include: ['./src/__tests__/**/*.test.jsx'],
     globals: true,
   },
+  resolve: {
+    alias: {
```
Comment (vatsalsinghkv) [inline vitest.config.ts:14]: @akartis-dev Why aren't we adding path in `tsconfig`

Comment (akartis-dev) [reply vitest.config.ts:14]: I already add path in tsconfig, you can verify in changed file

**Review by vatsalsinghkv — APPROVED** [review]
Nice work ❤️

**Pull request merged**